data/south jobs.txt

```diff
@@ -150,6 +153,7 @@ mission "Sketchy Cargo [0]"
 	on visit
 		dialog phrase "generic cargo on visit"
 	on complete
+		
```

Comment:
Did you forget to add something here, or is it just an accidentally added empty line?

Reply:
Good eye. Its an easy enough fix for later cleanup. We'll worry more about this sort of thing when we get closer to release.